src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days >= 7) {
+    return (sum += days * 40 - 50);
+  }
+
+  if (days >= 3) {
+    return (sum += days * 40 - 20);
+  }
+
+  if (days > 0) {
+    return (sum += days * 40);
```

Comment:
It might be a good idea not to use the same code in several places. Otherwise, everything is good 👍